src/shared/components/tableHeaderControls/TableHeaderControls.tsx

```diff
@@ -70,6 +77,8 @@ export default class TableExportButtons extends React.Component<ITableExportButt
         showPagination:false,
         searchClassName: '',
         copyDownloadClassName: '',
+        downloadFilename: 'patient-clinical-attributes.csv',
```

Comment:
@alisman `patient-clinical-attributes.csv` was hard-coded as the download file name value. I made it customizable and set the default value. But I still don't think it is a good idea to have `patient-clinical-attributes.csv` as a default value in this component. A generic name like `table.csv` might be better.